### ISSUE TITLE
URL Query problem with phalcon serve

### ISSUE THREAD

**Issue by AdnanBabakan** [phalcon/phalcon-devtools#1305]
### Expected and Actual Behavior

When using ``phalcon serve`` as a development serve when trying to have url query for GET parameters development server cannot recognize it correctly.

``http://icriticize.local/x/login?_m=SUCC_LOGOUT`` will cause the error below:
```
iCriticize\Controllers\IndexController handler class cannot be loaded

#0 [internal function]: Phalcon\Mvc\Dispatcher->_throwDispatchException('iCriticize\\Cont...', 2)
#1 [internal function]: Phalcon\Dispatcher->dispatch()
#2 C:\xampp\htdocs\icriticize\public\index.php(42): Phalcon\Mvc\Application->handle()
#3 C:\xampp\htdocs\icriticize\.htrouter.php(30): require_once('C:\\xampp\\htdocs...')
#4 {main}
```
and using the url ```http://icriticize.local/x/login/?_m=SUCC_LOGOUT``` with a ``/`` at the end of the route before the query will cause the error below:
```
iCriticize\Controllers\XController handler class cannot be loaded

#0 [internal function]: Phalcon\Mvc\Dispatcher->_throwDispatchException('iCriticize\\Cont...', 2)
#1 [internal function]: Phalcon\Dispatcher->dispatch()
#2 C:\xampp\htdocs\icriticize\public\index.php(42): Phalcon\Mvc\Application->handle()
#3 C:\xampp\htdocs\icriticize\.htrouter.php(30): require_once('C:\\xampp\\htdocs...')
#4 {main}
```
Totally means that without ``/`` at the end of the url causes phalcon to look for the default controller and with ``/`` at the end of the url causes phalcon to look for the correct controller but still cause error.

### Details

Phalcon info:
```
Phalcon DevTools (3.4.0)

Environment:
  OS: Windows NT ADNAN-PC 10.0 build 18362 (Windows 10) AMD64
  PHP Version: 7.3.5
  PHP SAPI: cli
  PHP Bin: C:\xampp\php\php.exe
  PHP Extension Dir: C:\php\ext
  PHP Bin Dir: C:\php
  Loaded PHP config: C:\xampp\php\php.ini
Versions:
  Phalcon DevTools Version: 3.4.0
  Phalcon Version: 3.4.3
  AdminLTE Version: 2.3.6
```

**Comment by Jeckerson** [comment]
Can you post here the routes that handles `/x/login?_m=SUCC_LOGOUT`?

**Comment by AdnanBabakan** [comment]
Here you are:
```php
$x_user->add('/login', ['action' => 'login'])->setName('login');
```

**Comment by CameronHall** [comment]
@Ultimater would this have anything to do with the changes you've made?

**Comment by Ultimater** [comment]
@CameronHall Negative. The changes you speak of I had Jackerson revert so I wouldn't be holding back the 3.4.2 release, the version being used here is 3.4.0 and thus wouldn't have been affected anyways.

@AdnanBabakan Are you able to edit your index.php file and compare the output of the following between Apache and `phalcon serve` to see if you notice any differences?

```
<?php
echo $_GET['_url'];
exit;
?>
```

**Comment by Jeckerson** [comment]
Fixed in `3.4.7`. Feel free to re-open in case new occurrences.
